mapdata/WarcraftLegacies/AbilityData/Skin/1297035329.json

```diff
@@ -8,16 +8,22 @@
     {
       "Id": 1718840929,
       "Type": 3,
-      "Value": "(Twilight DK)"
+      "Value": "(Scourge DK)"
```

Comment:
You need to update the Death Knight tooltip with this new ability. It still says Black Strike.

Reply:
Changed